.github/workflows/foam-vscode.yml

```diff
@@ -0,0 +1,29 @@
+name: Test foam-vscode
+
+on:
+  pull_request:
+    paths:
+      - 'packages/foam-vscode/**'
+  push:
+    paths:
+      - 'packages/foam-vscode/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Lint foam-vscode
+        run: yarn workspace foam-vscode lint
+      # - name: Test foam-vscode
+      #   run: yarn workspace foam-vscode test
+      # - name: Publish foam-vscode
+      #   if: github.ref == 'refs/heads/master'
+      #   run: yarn workspace foam-vscode publish-extension
+      #   with:
+      #     vsce_token: ${{ secrets.VSCE_TOKEN }}
```

Comment:
I assume this is commented out, but left in intentionally because we'll soon be using it?

Reply:
Yeah I'm thinking it could be a good idea to publish it automatically when it lands on master

Reply:
That would be amazing! (I've been waiting for the Daily Notes feature 😅)

Reply:
@mathieudutour @jsjoeio agreed, auto deploy from master would be dope. I'm not 100% sure how I feel about that without having any kind of vs code e2e test coverage -- maybe we could autopublish to an alpha channel for the brave (under different extension id), and then do stable releases ~weekly?

Reply:
Yeah that sounds good

Reply:
Sounds good to me too